operator/config/manager/manager.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: ctrlman
```

Comment:
if its the manifest operator, we could call it manifest-manager maybe? just a suggestion im fine with either

Reply:
I just wanted the name to be as short as possible. It's in the `manifest-operator` namespace anyway. If we insist on a correct wording, then maybe just `manager` is enough.